package/Dockerfile.submariner-diagnose

```diff
@@ -0,0 +1,8 @@
+FROM scratch
+
+ENV USER_UID=1001 PATH=/
+
+# install subctl binary
+COPY bin/subctl /subctl
```

Comment:
Since this is essentially packaging `subctl` shouldn't the image be just named `submariner-subctl` or event just `subctl`?

Reply:
Yes, I was going to name it `submariner-subctl` too but wasn't sure at the time if we will need anything more than subctl going forward for diagnostics. Am okay with either, we can always revisit this if need be in future. `submariner-subctl` or `subctl` ?

Reply:
IDK we already have `nettest` I think in that light we can call it simply `subctl` but I'm fine either way, whatever seems most logical to everyone else